src/execution_tree/primitives/base_primitives.cpp

```diff
@@ -1195,7 +1235,7 @@ namespace phylanx { namespace execution_tree
         case 0:
             return ast::nil{};
 
-        case 1:     // bool
+        case 1:    // phylanx::ir::node_data<bool>
```

Comment:
I think that this switch has to be left alone as it looks at the variant stored in an AST.